cross/fossil/Makefile

```diff
@@ -0,0 +1,28 @@
+PKG_NAME = fossil-scm
```

Comment:
it is common to name the folder like the package where applicable (use cross/fossil-scm  **and spk/fossil-scm**)